benchmark/kernels/fused_moe_triton/tuning_fused_moe_triton.py

```diff
@@ -393,7 +393,7 @@ def main(args: argparse.Namespace):
         topk = config.num_experts_per_tok
         intermediate_size = config.intermediate_size
         shard_intermediate_size = 2 * intermediate_size // args.tp_size
-    elif config.architectures[0] == "Qwen2MoeForCausalLM":
+    elif config.architectures[0] in ["Qwen2MoeForCausalLM", "Qwen3MoeForCausalLM"]:
```

Comment:
After this, may we delete
```python
    elif config.architectures[0] == "Qwen3MoeForCausalLM":
        E = config.num_experts
        topk = config.num_experts_per_tok
        intermediate_size = config.moe_intermediate_size
        shard_intermediate_size = 2 * intermediate_size // args.tp_size
```

Reply:
https://github.com/sgl-project/sglang/blob/74cb12a8786f17ba03378838f11b20d0d74b75e7/benchmark/kernels/fused_moe_triton/tuning_fused_moe_triton.py#L401-L405